src/sentry/api/bases/project.py

```diff
@@ -199,7 +207,7 @@ def get_filter_params(self, request: Request, project, date_filter_optional=Fals
 
         return params
 
-    def handle_exception(
+    def handle_exception(  # type: ignore[override]
```

Comment:
I plan to unwind this separately -- we're clobbering a rest-framework method with an utterly-incompatible signature